akka-docs/src/main/paradox/scala/additional/leftovers.md

```diff
@@ -1,57 +1,66 @@
-# Getting Started
+# TODO
```

Comment:
placed some stuff from the old intro here until we decide where to put it or throw it away